phoenix-core/src/main/java/org/apache/phoenix/mapreduce/PhoenixServerBuildIndexInputFormat.java

```diff
@@ -85,14 +89,17 @@ protected  QueryPlan getQueryPlan(final JobContext context, final Configuration
             Long scn = (currentScnValue != null) ? Long.valueOf(currentScnValue) : EnvironmentEdgeManager.currentTimeMillis();
             configuration.set(PhoenixConfigurationUtil.CURRENT_SCN_VALUE,
                     Long.toString(scn));
+
```

Comment:
We should also change the line 89 above and use the current time - 60 seconds  instead of using the current time for setting scn (as we do in IndexScrutiny). Then we need to sleep 60 second in the index tool before starting the MR job. If you prefer to do it in a different jira, that is also fine with me.

Reply:
Good idea @kadirozde! Looks like a small enough change to include in this change.

Reply:
I would prefer new Jira as it needs a better review covering every scenario and lots of ground for testing.